Big-Spender/readme.md

```diff
@@ -127,6 +143,14 @@ CREATE YOUR QUERY HERE
 INSERT YOUR QUERIES HERE
 
 ```
+INSERT INTO expense_types (expense_type) VALUES ('Hardware');
+INSERT INTO expense_areas (expense_area) VALUES ('IT');
+INSERT INTO suppliers (supplier) VALUES ('Dell');
+INSERT INTO spends (expense_type_id, expense_area_id, supplier_id, date, transaction_no, supplier_inv_no, description, amount) VALUES (42, 46, 66,'19-08-2021', 38104091, 3780119655, 'Computer Hardware Dell', 32000);
+
```

Comment:
Good job with creating the insert into statements for each of the types (expense_types, expense_areas, suppliers) before using the insert statement for spends.